skrub/_gap_encoder.py

```diff
@@ -724,14 +724,14 @@ class GapEncoder(TransformerMixin, BaseEstimator):
     activation of each topic for each category:
 
     >>> enc.transform(X)
-    array([[ 0.05202843, 10.54797156],
-          [ 0.05000118,  4.54999882],
-          [12.04734788,  0.05265212],
-          [ 0.05263068, 16.54736932],
-          [ 6.04999624,  0.05000376],
-          [19.546716  ,  0.053284  ],
-          [ 6.04999623,  0.05000376],
-          [ 0.05002016,  4.54997983]])
+    array([[ 0.05181645, 10.54818355],
+           [ 0.05013363,  4.54986637],
+           [12.04651708,  0.05348292],
+           [ 0.05227038, 16.54772961],
+           [ 6.04996972,  0.05003028],
+           [19.54522669,  0.0547733 ],
+           [ 6.04996972,  0.05003028],
+           [ 0.06011992,  4.53988008]])
```

Comment:
I would use some ellipsis to avoid future problem here.
```suggestion
    array([[ 0.051..., 10.548...],
           [ 0.050...,  4.549...],
           [12.046...,  0.053...],
           [ 0.052..., 16.547...],
           [ 6.049...,  0.050...],
           [19.545...,  0.054...],
           [ 6.049...,  0.050...],
           [ 0.060...,  4.539...]])
```